dask/dataframe/io/parquet/core.py

```diff
@@ -453,7 +456,7 @@ def to_parquet(
         )
 
     if compute:
-        out = out.compute()
+        out = out.compute(**compute_kwargs)
```

Comment:
If `compute_kwargs` is still `None` here then the `**` unpacking will fail, so we'll need a check above this to make it an empty-dict if it hasn't been specified (or to not pass it if it's `None`)